packages/flutter_test/lib/src/controller.dart

```diff
@@ -779,41 +810,142 @@ abstract class WidgetController {
     });
   }
 
+  /// Calls [debugPrint] with the given message.
+  ///
+  /// This is overridden by the WidgetTester subclass to use the test binding's
+  /// [TestWidgetsFlutterBinding.debugPrintOverride], so that it appears on the
+  /// console even if the test is logging output from the application.
+  @protected
+  void printToConsole(String message) {
+    debugPrint(message);
+  }
+
   // GEOMETRY
 
   /// Returns the point at the center of the given widget.
-  Offset getCenter(Finder finder) {
-    return _getElementPoint(finder, (Size size) => size.center(Offset.zero));
+  ///
+  /// If `warnIfMissed` is true (the default is false), then the
+  /// returned coordinate is checked to see if it would actually hit
```

Comment:
This doc is confusing. getCenter doesn't actually hit anything?

Maybe the parameter name is even confusing? What is there to miss when you get the center of a thing?

Reply:
Maybe the public getCenter, getTopLeft and friends methods shouldn't expose these new parameters? Instead tap and friends and getCenter could both call to a private _getCenterWithWarning method?

Reply:
Adjusted the documentation to be clearer.

I do use the new argument on some calls to getCenter, and probably should also do so on some calls to the others, but I got tired of porting tests over. :-)